### PR TITLE
Add missing schedules where guided_partitioning applies 

### PR DESCRIPTION
**Pull request by JRivrain**
See https://progress.opensuse.org/issues/93641

Vrs: 
ext3_textmode http://waaa-amazing.suse.cz/tests/15546
ext4_textmode http://waaa-amazing.suse.cz/tests/15545
xfs_textmode http://waaa-amazing.suse.cz/tests/15549

needle: https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/732

### CODE REVIEW

**Comment by jknphy** [comment]
I would suggest to change the title of this PR to something that reflect better the work:
So instead of `Add a schedule file for several test suites` we could have `Add schedules for guided_partitioning in O3` if you would add something extra, it would be still the main point.
Another observation is that for the commit it doesn't make sense to add the verifications, it is hidden and the git history is for reading it much more later in time.

**Comment by jknphy** [comment]
Regarding with you selection, I think it is a good choice, we will not switch to libyui for those test suite that we don't have in SLE (or that somehow looks like belong to us). I would only add btrfs+warnings to your list.

File: schedule/yast/btrfs/btrfs_opensuse.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@ name:           btrfs
 description:    >
   Validate default installation with btrfs.
 vars:
-  FILESYSTEM: btrfs
```
Comment (jknphy) [inline schedule/yast/btrfs/btrfs_opensuse.yaml:5]: we still cannot remove it here, right? https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/4357ef1a9764fa73e8074c02dd2019bd15fc1957/lib/y2_base.pm#L89

Comment (JRivrain) [reply schedule/yast/btrfs/btrfs_opensuse.yaml:5]: good catch ! I'll have a look.

Comment (JRivrain) [reply schedule/yast/btrfs/btrfs_opensuse.yaml:5]: I'm not so sure about that one. Collecting those btrfs-specific logs can be useful, but in this form is not really useful. If we want to keep it, it makes sense to completely revert the commit and keep that variable.

**Comment by JRivrain** [comment]
> Regarding with you selection, I think it is a good choice, we will not switch to libyui for those test suite that we don't have in SLE (or those which don't look somehow like belonging to us). I would only add btrfs+warnings to your list.

btrfs+warnings already has a schedule file.

File: schedule/yast/btrfs/btrfs_opensuse.yaml (hunk 1 above)
Comment (JRivrain) [reply schedule/yast/btrfs/btrfs_opensuse.yaml:5]: I kept the variable here in the end.

Comment (JRivrain) [reply schedule/yast/btrfs/btrfs_opensuse.yaml:5]: It looks like this variable is only used in this test suite.

File: schedule/yast/btrfs/btrfs_opensuse.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name:           btrfs
...
+  FILESYSTEM: btrfs  # Used for log collection in lib/y2_base
```
Comment (jknphy) [inline schedule/yast/btrfs/btrfs_opensuse.yaml:5]: I'm not sure about the parser with comments on the rights, I guess it is fine but might give troubles sometimes, it is better on top of the line. Better way would be to create a ticket to remove this dependency  and being able to check if the filesystem is btrfs and avoid the comment. We could refine that of course.

**Comment by jknphy** [comment]
Please, restart all test suites changed in O3.

**Pull request merged**